libs/components/datetime/src/lib/modules/datepicker/datepicker.component.spec.ts

```diff
@@ -1253,6 +1253,25 @@ describe('datepicker', () => {
         expect(getInputElementValue(fixture)).toBe('05/12/2017');
         expect(component.dateControl.value).toEqual(new Date('5/12/2017'));
       }));
+
+      it('should set the initial value correctly if the form is disabled on creation and then reenabled', fakeAsync(() => {
+        fixture = TestBed.createComponent(DatepickerReactiveTestComponent);
+        nativeElement = fixture.nativeElement as HTMLElement;
+        component = fixture.componentInstance;
+
+        // Default to US long date format to avoid any test runners that are using a different locale.
+        component.dateFormat = 'MM/DD/YYYY';
+
+        component.initialValue = '5/12/2017';
+        component.disableFormOnCreation = true;
+        detectChanges(fixture);
+
```

Comment:
So that is where the issue comes in. I agree...... but we don't have a reference to the control internally to the input directive at the point that `writeValue` is called to update the control's value with the converted date. If you have ideas as to how to get this - I'm all ears :). But without that we have to wait until we have that reference which is in the `validate` function....

Reply:
Actually, I found that some of our other components are grabbing the control a different way. Let me see if I can quickly convert datepicker to do this to solve the issue here.

Reply:
After some trial and error and discussions with @Blackbaud-SteveBrush I have updated the input directive to grab the control via the injector when needed and am utilizing a set timeout to address the inner "AbstractControl" being undefined if the control is initially disabled when `writeValue` is called from Angular.

Reply:
I reverted the previously mentioned change as it caused a race condition between when `writeValue` fully succeeded and the `validate` method.